### DISCUSSION TITLE
Icon for Google scholar missing

### DISCUSSION THREAD

**Discussion by AntoineSoetewey** [jpanther/congo#717] [Q&A] [answered]
Hello,

When looking at all the available icons (https://jpanther.github.io/congo/samples/icons/), I am wondering if there is a reason why the one for Google Scholar is missing?

Is it linked to this thread https://github.com/FortAwesome/Font-Awesome/issues/541#issuecomment-1824556786?

Regards,
Antoine

**Comment by jpanther** [comment, marked as answer]
It is mainly because it's not available in FontAwesome, yes. But if you have a need for a custom icon it's very easy to add it yourself so I haven't seen this as a big issue.

**Comment by AntoineSoetewey** [comment]
It seems that they have added the Google scholar icon in FontAwesome: https://github.com/FortAwesome/Font-Awesome/issues/541#issuecomment-1830481944

**Reply by jpanther** [reply]
So they have! I'll create an issue for this so it can be added in at some point in the near future.
